### PR TITLE
#161779731  Setup Redux

### PR DESCRIPTION
**Pull request by chinemelu**
#### What does this PR do?
Setup React-Redux folder/file structure

#### Description of Task to be completed?
- Create appropriate files and folders
- Populate files with dummy data
- Install necessary dependencies 

#### How should this be manually tested?
N/A

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#161779731](https://www.pivotaltracker.com/story/show/161779731)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,14 +21,18 @@
...
     "react-dom": "16.7.0",
-    "react-router-dom": "4.3.1"
+    "react-redux": "6.0.0",
+    "redux": "4.0.1",
+    "redux-thunk": "2.3.0"
   },
   "devDependencies": {
+    "babel-plugin-transform-object-rest-spread": "6.26.0",
+    "html-webpack-plugin": "3.2.0",
+    "react-router-dom": "4.3.1",
```
Comment (Daymorelah) [inline package.json]: I think react-router-dom package should be in the dependencies and not in the devDependencies.

**Review by Daymorelah — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (kingisaac95) [reply package.json]: Yes please

**Review by Daymorelah — DISMISSED** [review]

File: src/actionTypes/index.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+export const EXAMPLE = 'EXAMPLE';
+export const EXAMPLETWO = 'EXAMPLE-TWO'
```
Comment (kingisaac95) [inline src/actionTypes/index.js]: Newline here please

File: src/components/StartPage.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    super(props)
+
+    this.state = {
+
+    }
+  }
+
+  render() {
+    return (
+      <h1>Hello World, this is Redux, baby!</h1>
```
Comment (kingisaac95) [inline src/components/StartPage.js:18]: 😄

File: src/reducers/index.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+
+import { combineReducers } from 'redux';
+import reducers from './reducers';
+
+
+const rootReducers = combineReducers({
+  reducers
+});
+
+export default rootReducers;
```
Comment (kingisaac95) [inline src/reducers/index.js]: Newline here

File: src/reducers/index.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+
+import { combineReducers } from 'redux';
+import reducers from './reducers';
+
+
+const rootReducers = combineReducers({
+  reducers
```
Comment (kingisaac95) [inline src/reducers/index.js]: Breaking this into multiple lines is unnecessary

File: src/reducers/reducers.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+    case EXAMPLETWO: 
+      return {
+        ...state,
+        examplePropTwo: action.payload
+      }
+    default: return state;
+  }
+}
+
+export default reducers;
```
Comment (kingisaac95) [inline src/reducers/reducers.js]: Newline here

File: src/components/App.js [hunk 7]
```diff
@@ -1,14 +1,14 @@
...
+     <div>
+       {this.props.children}
+       <h3 className="test2">Authors Haven</h3>
+     </div>
+   );
+ }
 }
 
-export default App;
+export default App;
```
Comment (kingisaac95) [inline src/components/App.js]: Newline here.
Why did you change the tab size here? It should be consistent with all other files.

**Review by kingisaac95 — APPROVED** [review]
